roles/cv-greeter-theme/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Install Greeter Dependencies
+  xbps:
+    pkg:
+      - git-all
+      - ImageMagick
+    state: present
+
+- name: Deploy cv-greeter-theme
+  git:
+    repo: https://github.com/collegiumv/cv_greeter_theme.git
+    dest: /usr/share/lightdm-webkit/themes/cv
+    version: december
+    force: yes
+
+- name: Force mode on theme folder
+  file:
+    path: /usr/share/lightdm-webkit/themes/cv
+    state: directory
+    recurse: yes
+    owner: root
+    group: root
+    mode: ugo+rX
+
+- name: Download assets
+  get_url:
+    url: "{{ item }}"
+    dest: /usr/share/lightdm-webkit/themes/cv/
+    owner: root
+    group: root
+    mode: 0644
+  with_items:
+    - http://code.jquery.com/jquery-2.1.4.min.js
+    - http://upload.wikimedia.org/wikipedia/commons/c/c6/VISTA%27s_infrared_view_of_the_Orion_Nebula.jpg
+
+- name: Scale Wallpaper down
+  command: convert /usr/share/lightdm-webkit/themes/cv/VISTA%27s_infrared_view_of_the_Orion_Nebula.jpg -resize 3000x3000 -rotate 90 /usr/share/lightdm-webkit/themes/cv/wallpaper.jpg
```

Comment:
Is there a better way to convert here?  Blindly resizing 3000x3000 seems like a poor idea.  Imagemagick provides some flags for resize ([\\>](http://www.imagemagick.org/Usage/resize/#shrink), [^](http://www.imagemagick.org/Usage/resize/#fill)) that we could look into, along with xrandr, to resize and crop to the most optimal dimensions.  This is especially nice because webkit-resizing is not optimal at all.

Might be a problem for another commit, though.